src/components/Select/Select.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useMemo,useRef, useState } from 'react';
+
+import { Icon } from '../Icon';
+
+import {
+  SelectButtonWraper,
+  SelectItem,
+  SelectLabel,
+  SelectList,
+  SelectWraper,
+} from './styled';
+import { SelectInterface } from '.';
+
+export const Select = ({
+  label,
+  placeholder,
+  options = [],
+  value,
+  onChange,
+}: SelectInterface) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const selectRef = useRef<HTMLDivElement>(null);
+
+  const handleClickOutside = (event: MouseEvent) => {
+    if (
+      selectRef.current &&
+      !selectRef.current.contains(event.target as Node)
+    ) {
+      setIsOpen(false);
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener('mousedown', handleClickOutside);
```

Comment:
set blank line before `return` statement

Reply:
Accepted and fix